kafka-connect-s3/docs/s3_connector.rst

```diff
@@ -314,3 +314,87 @@ schema generator class to its default value.
 Finally, schema evolution is disabled in this example by setting ``schema.compatibility`` to ``NONE``, as explained above.
 
 
+Write raw message values into S3
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+It is possible to use the S3 connector to write out the unmodified original message values into
+newline-separated files in S3. We accomplish this by telling Connect to not deserialize any of the
+messages, and by configuring the S3 connector to store the message values in a binary format in S3.
+
+The first part of our S3 connector is similar to other examples:
+
+.. sourcecode:: bash
+
+  name=s3-raw-sink
+  connector.class=io.confluent.connect.s3.S3SinkConnector
+  tasks.max=1
+  topics=s3_topic
+  flush.size=3
+
+The ``topics`` setting specifies the topics we want to export data from, in this case ``s3_topic``.
+The property ``flush.size`` specifies the number of records per partition the connector needs
+to write before completing a multipart upload to S3.
+
+Next we need to configure the particulars of Amazon S3:
+
+.. sourcecode:: bash
+
+  s3.bucket.name=confluent-kafka-connect-s3-testing
+  s3.region=us-west-2
+  s3.part.size=5242880
+  s3.compression.type=gzip
+
+The ``s3.bucket.name`` is mandatory and names your S3 bucket where the exported Kafka records
+should be written. Other useful settings are ``s3.region``, which you should set if you use a
+region other than the default, and ``s3.part.size`` to control the size of each part in the
+multipart uploads that will be used to upload a single chunk of Kafka records.
+The ``s3.compression.type`` specifies that we want the S3 connector to compress our S3 objects
+using GZIP compression, adding the ``.gzip`` extension to any files (see below).
```

Comment:
`gzip` is the correct name for the `s3.compression.type` value, but the extension added to files is `.gz` ([second argument to CompressionType](https://github.com/confluentinc/kafka-connect-storage-cloud/blob/v4.1.0-rc3/kafka-connect-s3/src/main/java/io/confluent/connect/s3/storage/CompressionType.java#L39)).